templates/zerver/help/communities-directory.md

```diff
@@ -2,24 +2,38 @@
 
 {!communities-directory-intro.md!}
 
+## Eligibility
+
 In order to be eligible for listing in the communities directory, an
-organization must be open to the public. Your organization is eligible if users are
-allowed [join without an invitation][join-without-invite] and/or the [public
-access option](/help/public-access-option) is enabled.
+organization must be open to the public. Your organization is eligible
+if users are allowed [join without an invitation][join-without-invite]
```

Comment:
Do we mean "allowed to join without ..."? This seems to be a preexisting typo.

Reply:
Yes. Thanks for catching that!